src/scriptdev2/scripts/eastern_kingdoms/shadowfang_keep/shadowfang_keep.cpp

```diff
@@ -598,8 +598,11 @@ struct boss_arugalAI : public ScriptedAI
         {
             if (!m_bAttacking)
             {
-                DoCastSpellIfCan(m_creature->getVictim(), SPELL_VOID_BOLT);
-                m_uiVoidboltTimer = urand(2900, 4800);
+                if (Unit* victim = m_creature->getVictim())
```

Comment:
Eventually we can do another redundant check for "pTarget" in this function: 
https://github.com/cmangos/mangos-wotlk/blob/master/src/game/CreatureAI.cpp#L84

So then it will be clear for any spell cast from script.